niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/log/UiLogger.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2020 The Android Open Source Project
```

Comment:
Ultra-nit: copyright year should be set to the year this file was created.

Reply:
Updated.